pkg/cmd/sponsors/list/list_test.go

```diff
@@ -0,0 +1,127 @@
+package list
```

Comment:
A bit of personal preference here but I would like to see us generally moving towards using `list_test` packages so that we have to work through public interfaces as the default.

Reply:
Let me repeat back to you what I think this means:

1. The tests for a given package should be their own package, called `package_test`
2. The package that's being tested would then be imported and referenced by `package.*`
3. This enforces the interfaces that are being tested to be public by default, because if they weren't we wouldn't be able to call them in the test suite

If this is correct, then I guess my question is why default to public interfaces for everything being tested? I've generally been of the opinion that public interfaces should only be provided for what is required for other packages. In this case, that means that the `NewCmdList` is the only thing that's public, because it's the only thing needed outside of the `list` package functionality. All the additional functions I'm testing are only their own functions to make testing easier and code more readable, so why make them public by default?